contrib/auto_gdb/README.md

```diff
@@ -0,0 +1,45 @@
+# Contents
+This directory contains tools to automatically get data about the memory consumption by some objects in dashd process with the help of GDB debugger.
+
+## dash_dbg.sh
+This shell script attaches GDB to running dashd process (should be built with debug info), executes debug.gdb script and detaches.
+By default it uses testnet setting,  see script comments to attach to mainnet dashd.
+
+## debug.gdb
+Contains debugger instructions to execute during attach: load python code and executes it for the objects we want to investigate.
+
+## log_size.py
+Contains definition of the gdb command log_size. After loading of this script it can be called from gdb command line or other gdb scripts.
```

Comment:
```suggestion
Contains definition of the gdb command log_size. After this script loads it can be called from gdb command line or other gdb scripts.
```